src/constants/editors.ts

```diff
@@ -6,24 +6,16 @@ import Chrome from "../editors/chrome";
 import Eclipse from "../editors/eclipse";
 import AndroidStudio from "../editors/android-studio";
 import VSCode from "../editors/vscode";
+import XCode from "../editors/xcode";
 
-export const ANDROIDSTUDIO = "ANDROIDSTUDIO";
-export const ATOM = "ATOM";
-export const CHROME = "CHROME";
-export const ECLIPSE = "ECLIPSE";
-export const SUBLIMETEXT2 = "SUBLIMETEXT2";
-export const SUBLIMETEXT3 = "SUBLIMETEXT3";
-export const VIM = "VIM";
-export const VSCODE = "VSCODE";
-export const XCODE = "XCODE";
-
-export default [
-  { name: ATOM, instance: new Atom() },
-  { name: CHROME, instance: new Chrome() },
-  { name: ECLIPSE, instance: new Eclipse() },
-  { name: ANDROIDSTUDIO, instance: new AndroidStudio() },
-  { name: SUBLIMETEXT2, instance: new SublimeText2() },
-  { name: SUBLIMETEXT3, instance: new SublimeText3() },
-  { name: VIM, instance: new Vim() },
-  { name: VSCODE, instance: new VSCode() }
-];
+export default {
+  Atom,
+  Chrome,
+  Eclipse,
+  AndroidStudio,
+  SublimeText2,
+  SublimeText3,
+  Vim,
+  VSCode,
+  XCode
+};
```

Comment:
There're more editors ready to be added into thsi list. Could you please add them?

Reply:
yes, let me add all of them 👍

Reply:
done